.github/workflows/release.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         include:
```

Comment:
We should entirely remove the `release` CI file - it's the one that packages things on release, but we won't be doing that anymore